clients/fides-js/src/components/fides.css

```diff
@@ -1118,8 +1100,6 @@ button.fides-banner-button.fides-menu-item:not([aria-pressed="true"]):hover {
   }
 
   div#fides-banner-inner-container {
-    display: flex;
-    flex-direction: column;
```

Comment:
Can you explain why these CSS changes were needed or generally what these changes do?

We're also making some CSS changes here that we need to eval to see if there are any conflicts- [clients/fides-js-extensions/nytimes/src/ext.css](https://github.com/ethyca/fidesplus/pull/1582/files)

Reply:
all CSS changes are being made as part of this layout update ticket https://ethyca.atlassian.net/browse/PROD-2611 which is pretty heavily tied to the other TCF updates so I'm including them in this PR. All of these affect TCF only, and they affect the banner only. No style updates for modal at all and no updates for non-TCF banner.